### PR TITLE
Added a couple methods to the NMSAssistant class.

### PR DESCRIPTION
**Pull request by dmccoystephenson**
# Purpose
The changes in this PR will allow developers to check whether the NMS version is **greater than or equal to ( >= )** or **less than or equal to ( <= )**.

## Notes
- The change to the functionality of the isVersionGreaterThan() method in #102 has been reverted to avoid misleading developers.

### CODE REVIEW

File: src/main/java/preponderous/ponder/minecraft/bukkit/nms/NMSAssistant.java [hunk 1]
```diff
@@ -26,6 +24,15 @@ public NMSVersion getNMSVersion() {
...
     public boolean isVersionGreaterThan(int version) {
+        return getNMSVersion().getMinor() > version;
+    }
+
+    /**
+     * Method to determine if the server version is greater than or equal to the given version.
+     * @param version to test against.
+     * @return {@code true} if it is.
+     */
+    public boolean isVersionGreaterThanOrEqualTo(int version) {
```
Comment (VoChiDanh) [inline src/main/java/preponderous/ponder/minecraft/bukkit/nms/NMSAssistant.java:35]: that's prefect, thank you so much

**Comment by dmccoystephenson** [comment]
Hey @VoChiDanh I added you as a reviewer to this PR because you recently opened #102. Let me know if you approve these changes.

**Comment by dmccoystephenson** [comment]
Hey @CallumJohnson I added you as a reviewer because you originally contributed these files. Don't worry about responding if you don't have time :)

File: src/main/java/preponderous/ponder/minecraft/bukkit/nms/NMSAssistant.java (hunk 1 above)
Comment (dmccoystephenson) [reply src/main/java/preponderous/ponder/minecraft/bukkit/nms/NMSAssistant.java:35]: That's awkward I didn't see your response here lol

**Pull request merged**